configs/config_all.yaml

```diff
@@ -122,6 +122,8 @@ process:
       cv_classifier: ''                                       # OpenCV classifier path for face detection. By default, we will use 'haarcascade_frontalface_alt.xml'.
       blur_type: 'gaussian'                                   # type of blur kernel, including ['mean', 'box', 'gaussian']
       radius: 2                                               # radius of blur kernel
+  - image_tagging_mapper:                                   # Mapper to generate image tags.
+      tag_field_name: '__dj__image_tags__'                    # the field name to store the tags. It's "__dj__image_tags__" in default.
```

Comment:
Just a thought: many built-in `constants` mix single and double underscores, which may be confusing. Is it necessary to standardize them to single underscores?

Reply:
The rule is kinda simple:
1. special word "dj" should be surrounded by "__" (two underscores)
2. the whole field name should be surrounded by "__" (two underscores)
3. words in the field name can be separated by "_" (one underscore)

I think it's better to keep still because this change will make it incompatible with the processed datasets before.